src/lib/services/ghost.service.ts

```
@@ -90,3 +90,115 @@ export const fetchGhostArticlesFilters = async (categories: string[], page = 1,
 		throw error;
 	}
 };
+
+export const createGhostMember = async (email: string, name: string = '', subscribedToEmails: boolean = true) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)